### ISSUE TITLE
Variable substitution not working

### ISSUE THREAD

**Issue by RyanJField** [FAIRDataPipeline/FAIR-CLI#213]
Config variable substitution for ${{DATETIME}} does not work  if used more than once in a config file, see the following workflow:
https://github.com/FAIRDataPipeline/rSimpleModel/runs/4980673879?check_suite_focus=true#step:16:107

**Comment by RyanJField** [comment]
@bruvio could you look into this and test the other variable substitutions please.

**Comment by RyanJField** [comment]
File example for yaml:
https://github.com/FAIRDataPipeline/rSimpleModel/blob/main/inst/extdata/SEINRDconfig.yaml

What the file should look like with variable substitution:
https://github.com/FAIRDataPipeline/rSimpleModel/commit/e6500c70bbea125b39475d8151a0a1a234a5fbf2

**Comment by bruvio** [comment]
@RyanJField  can you tell me how to reproduce this?
for example how do I test the creation of this config file? I assume some command from the cli have to be run.

**Comment by RyanJField** [comment]
Hi Bruno, if you make a user generated yaml containing:
```
 run_metadata:
  default_input_namespace: Testing
  description: Variable Testing
  script: 'echo ${{DATETIME}} ${{DATETIME}}'
```
Name it something like `test.yaml`
and run:
```
fair run test.yaml
```
From a virtual environment with an initialized fair directory (`fair init`)

**Comment by bruvio** [comment]
this is fixed in branch hotfix/fix-datetime-substitution](https://github.com/FAIRDataPipeline/FAIR-CLI/tree/hotfix/fix-datetime-substitution) 

as tests are broken I just tested using this approach https://github.com/FAIRDataPipeline/FAIR-CLI/issues/213#issuecomment-1056954899

@RyanJField please review and if ok we can merge

**Comment by bruvio** [comment]
this issue is fixed using fair-cli 0.3.0 (latest dev version)
